Simple optimization in IsKeyPressed

corfe83 opened the pull request:
This is my first change to ebiten. I will start with something small.

I noticed IsKeyPressed is iterating through every possible key to find the one being asked about. This change fixes it to only look at the key being asked about.

Obviously IsKeyPressed calls are not a big deal for performance, but it feels like the right thing to do.

hajimehoshi commented on genkeys.go:
```diff
@@ -342,6 +342,11 @@ var glfwKeyCodeToKey = map[glfw.Key]driver.Key{
 {{end}}
 }
+
+var keyCodeToglfwKey = map[driver.Key]glfw.Key{
```
`keyCodeToGLFWKey` sounds better

corfe83 replied:
Addressed in latest push

corfe83 commented:
> LGTM
> 
> Please update this PR by merging or rebasing, then I'll merge this PR.

Wow, quick response! Just rebased and force pushed this branch.

corfe83 commented:
> > LGTM
> > Please update this PR by merging or rebasing, then I'll merge this PR.
> 
> Wow, quick response! Just rebased and force pushed this branch.

Actually, something is off. Fixing.

corfe83 commented:
> > > LGTM
> > > Please update this PR by merging or rebasing, then I'll merge this PR.
> > 
> > 
> > Wow, quick response! Just rebased and force pushed this branch.
> 
> Actually, something is off. Fixing.

Fixed now.
